dist/linux/appimage/build.sh

```diff
@@ -77,11 +82,16 @@ mv jni/x86_64-Linux/* Cryptomator.AppDir/lib/app/libjffi.so
 rm -r jni/x86_64-Linux
 
 # load AppImageTool
-curl -L https://github.com/AppImage/AppImageKit/releases/download/13/appimagetool-x86_64.AppImage -o /tmp/appimagetool.AppImage
-chmod +x /tmp/appimagetool.AppImage
+curl -L https://github.com/AppImage/AppImageKit/releases/download/13/appimagetool-x86_64.AppImage -o appimagetool.AppImage
+chmod +x appimagetool.AppImage
+./appimagetool.AppImage --appimage-extract
```

Comment:
Extracting the app image was only needed in the environment of GitHub Actions. If AppImages work normally (which I assume if one wants to build one), there is no need for this step.